content/blog/2019-01-31-more-editors.md

```diff
@@ -0,0 +1,34 @@
+---
+slug: more_editors
+title: 'Announcing new software review editors: Melina Vidoni and Brooke Anderson'
+date: '2019-01-31'
+authors:
+  - Stefanie Butland
+categories: blog
+topicid:
+tags:
+  - Software Peer Review
+  - rOpenSci team
+---
+
+We are pleased to welcome Brooke Anderson and Melina Vidoni to our team of [Associate Editors](https://ropensci.github.io/dev_guide/softwarereviewintro.html#associateditors) for rOpenSci [Software Peer Review](https://ropensci.org/software-review/). They join Scott Chamberlain, Anna Krystalli, Lincoln Mullen, Karthik Ram, Noam Ross and Maëlle Salmon. With the addition of Brooke and Melina, our editorial board now includes four women and four men, located in North America, South America and Europe.
+
+Our open Software Peer Review system for community-contributed R tools is a key component of [our mission](https://ropensci.org/about/) to create technical infrastructure that lowers barriers to working with data sources on the web. Editors manage the review process, performing initial package checks, identifying reviewers, and moderating the process until the package is accepted by reviewers and transferred to rOpenSci. The number of packages submitted for review has increased every year, resulting in an increased [workload for Editors](https://ropensci.org/blog/2018/05/03/onboarding-is-work/#work-done-by-editors) and the need to expand the team.
+
+{{< figure src="/img/team/brooke-anderson.jpg" caption="Brooke Anderson" alt="Brooke Anderson" width="250" >}}
```

Comment:
I wonder whether `caption` is necessary here since the paragraph below starts with her name anyway?